recipes/ice-cream/build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -eu -o pipefail
+
+mkdir -p ${PREFIX}/bin
+cp ICEcream.sh ${PREFIX}/bin/ICEcream.sh
+cp ICEfamily_refer/familytools/plotting_script.py ${PREFIX}/bin/plotting_script.py
```

Comment:
I suspect you need to copy more than this, and also to change these shell scripts to refer to the scripts relative to the script's location (with `SCRIPT_DIR=$(dirname $(realpath $0))` or similar).

Specifically I'm talking about lines like

```
Rscript ICEfamily_refer/familytools/merge.version7.R 
```

AFAICT there is no way that whole familytools directory is copied into the package, just a couple of scripts. And even if it was, it wouldn't be in the current directory wherever the main script is run, and so the above will fail. You'd need `Rscript "${SCRIPT_DIR}/ICEfamily_refer/familytools/merge.version7.R" ...` with the above definition of `SCRIPT_DIR`

Reply:
Yes, currently this tool will fail a real test because of lacking relative paths. I will revise the code accordingly. Thanks for the suggestion!